### PR TITLE
[O2B-657] Store lhcPeriod value in runs

### PR DESCRIPTION
**Pull request by JikHuijberts**
#### I have a JIRA ticket
- [X] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [X] issue "Status" is set to "In review"
- [X] PR labels are selected
- [X] `CHANGELOG.md` and `database/CHANGELOG.md` files were updated

CHANGELOG will be updated with ticket O2B-627

Extra field is added within the run table:
`lhcPeriod` - string

This period can be updated by using the endpoint PATCH api/runs/:id

### CODE REVIEW

File: database/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## [0.3X.X]
```
Comment (graduta) [inline database/CHANGELOG.md:1]: Next release is due to be `0.35.0`

File: go-api-client/Example.go [hunk 2]
```diff
@@ -18,12 +18,12 @@ func main() {
 	// TODO: generate correct JWT token instead of manual insertion
 	apiClient.InitializeApi(baseUrl, apiToken)
 
-	// Create a run
+	// // Create a run
```
Comment (graduta) [inline go-api-client/Example.go:21]: No need for extra `//` here

File: go-api-client/Example.go [hunk 3]
```diff
@@ -18,12 +18,12 @@ func main() {
...
 	apiClient.CreateRun("go-api-Timestamp", 5, 5, 1, 80, sw.COSMICS_RunType, false, false, true, "normal",
 		(sw.CPV_Detectors + "," + sw.ZDC_Detectors + "," + sw.EMC_Detectors))
 
 	// Update a run
-	apiClient.UpdateRun(80, sw.BAD_RunQuality, -1, testTime, -1, testTime, false, false, "Repository Hash", "production/production.desc", "processing")
+	apiClient.UpdateRun(80, sw.BAD_RunQuality, -1, testTime, -1, testTime, false, false, "Repository Hash", "production/production.desc", "processing", "123123123")
```
Comment (graduta) [inline go-api-client/Example.go:26]: Could you please use as exmaple the same value that we are going to use in tests or a similar structured one?
`lhc22_b`

File: test/e2e/runs.test.js [hunk 4]
```diff
@@ -752,6 +752,7 @@ module.exports = () => {
                     timeO2End: dateValue,
                     timeTrgEnd: dateValue,
                     runQuality: 'test',
+                    lhcPeriod: '123123123',
```
Comment (graduta) [inline test/e2e/runs.test.js:755]: Please use the real like example `lhc22_b`

File: test/e2e/runs.test.js [hunk 5]
```diff
@@ -764,10 +765,11 @@ module.exports = () => {
                     expect(res.body.data.timeO2End).to.equal(dateValue);
                     expect(res.body.data.timeTrgEnd).to.equal(dateValue);
                     expect(res.body.data.runQuality).to.equal('test');
+                    expect(res.body.data.lhcPeriod).to.equal('123123123');
                     done();
                 });
         });
-        it('should be able to update with ', async () => {
+        it('should be able to update with lhc values', async () => {
```
Comment (graduta) [inline test/e2e/runs.test.js:772]: The description of the test is not entirely correct as it is updating a run with multiple details

**Review by graduta — CHANGES_REQUESTED** [review]

**Review by graduta — CHANGES_REQUESTED** [review]
Could you please add the LHC Period in the run-details page as well?
Probably best it would be under the other LHC information such as `BetaStar`

File: database/CHANGELOG.md (hunk 1 above)
Comment (JikHuijberts) [reply database/CHANGELOG.md]: Updated it to the right value

File: go-api-client/Example.go (hunk 2 above)
Comment (JikHuijberts) [reply go-api-client/Example.go]: I removed it now

File: go-api-client/Example.go (hunk 3 above)
Comment (JikHuijberts) [reply go-api-client/Example.go]: Yes missed that one!

File: test/e2e/runs.test.js (hunk 4 above)
Comment (JikHuijberts) [reply test/e2e/runs.test.js]: Same here

File: test/e2e/runs.test.js (hunk 5 above)
Comment (JikHuijberts) [reply test/e2e/runs.test.js]: Changed the name to `should be able to update with multiple run values`

**Comment by JikHuijberts** [comment]
> Could you please add the LHC Period in the run-details page as well? Probably best it would be under the other LHC information such as `BetaStar`

I added that in the next ticket: https://github.com/AliceO2Group/Bookkeeping/pull/669

**Review by graduta — APPROVED** [review]

**Pull request merged**